recipes/licensor/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,5 @@
+Package: licensor-0.4.3 (License: MIT)
+-----
+1 package licensed under BSD-2: text-1.2.4.0
+-----
+21 packages licensed under BSD-3: Cabal-3.0.1.0, array-0.5.4.0, base-4.13.0.0, binary-0.8.7.0, bytestring-0.10.10.0, containers-0.6.2.1, deepseq-1.4.4.0, directory-1.3.6.0, filepath-1.4.2.1, ghc-boot-th-8.8.3, ghc-prim-0.5.3, integer-gmp-1.0.2.0, mtl-2.2.2, parsec-3.1.14.0, pretty-1.1.3.6, process-1.6.8.0, rts-1.0, template-haskell-2.15.0.0, time-1.9.3, transformers-0.5.6.2, unix-2.7.2.2
```

Comment:
Thank for your review @isuruf. The whole point of this PR was to add a tool to conda-forge that allows us to add the dependency licenses for Haskell packages automatically. If this is not sufficient, then there is no point in merging this into conda-forge. I will ask the maintainer of licensor, whether copying all license files, instead of just summarising the license types, would be in scope for his project.

Reply:
@isuruf, pinging you again because the feedstock now contains all licences in the `DEPENDENCY_LICENSES.md` file. Would that satisfy the requirements for third party copyright notices?

Reply:
@isuruf, another ping. sorry to bother you with this, but it would be great to know if the third party notices are packages sufficiently now, so that I can get back to the author of `licensor` in [this PR](https://github.com/jpvillaisaza/licensor/issues/31).

What we're trying to do here is to make `licensor` the to-go tool for putting Haskell packages on conda-forge.

Reply:
Yes, looks good to me. Thanks for doing this work. Much appreciated.